esp/esp/program/models/class_.py

```diff
@@ -588,6 +588,18 @@ def already_passed(self):
             if time_passed > timedelta(0):
                 return True
         return False
+
+    def start_time_prefetchable(self):
+        """Like self.start_time(), but can be prefetched.
```

Comment:
In practice when this is being used, meeting_times should be prefetched, in which case getting it directly avoids a roundtrip to memcache, which is fairly quick since the memcache is on the same server, but is still nontrivial if you're doing it 50k times (as for example MIT Splash schedules do).  I'll update the comment, though.